### ISSUE TITLE
Show File Size before starting download

### ISSUE THREAD

**Issue by jsahil730** [subhra74/xdm#165]
## Checklist

- [x] I'm reporting a feature request
- [x] I've verified that I'm running xdm latest version
- [x] I've searched the bugtracker for similar feature requests including closed ones

## Description

WRITE DESCRIPTION HERE
I earlier used to work with eagleget on windows, but then switched over to xdman since I started using Ubuntu. Eagleget has this nice feature that it shows the file size before beginning the download, it is a nice to have so that one(probably they're running out of (mobile) data, so it would be not possible to download very large files) could not begin the download right now, but postpone it for sometime later. One could technically begin the download and then stop it, but this is a nice to have.

**Comment by subhra74** [comment]
It's basically same, to show the file size, request must be sent, so anyway some data is wasted anyway. So you can start the download and after starting the download you can always check the download size and stop the download. The other case is more wasteful because the program is already starting the download in background without any consent from the user.

**Comment by jsahil730** [comment]
@subhra74 I think a HEAD request would do the job if I am not wrong?
